doc/queries.md

```diff
@@ -35,6 +35,12 @@ Example query:
     ethnicity
     vital_status
     file_count
+    experiment {
```

Comment:
Can you also put the example result accordingly? the request and result should match.